src/main/webapp/sass/_core.scss

```diff
@@ -2,7 +2,7 @@
 Synapse color scheme
 ****************/
 $synapse-green: #469285;
-$synapse-blue: #2E4761;
+$synapse-blue: #395979;
```

Comment:
Small change from design (in general they want a darker blue for the primary color).